collect/collect.go

```diff
@@ -554,10 +554,10 @@ func (i *InMemCollector) redistributeTraces(ctx context.Context) {
 
 			sp.APIHost = newTarget.GetAddress()
 
-			if v := sp.Data.Get("meta.refinery.forwarded"); v != nil {
-				sp.Data.Set("meta.refinery.forwarded", fmt.Sprintf("%s,%s", v, i.hostname))
+			if sp.Data.MetaRefineryForwarded != "" {
```

Comment:
I think it's odd that some properties use the accessors, eg .Value and .Get(), .Set, but others do not. I think it would be best if we consolidate onto a single pattern.

Reply:
Yes! we have a follow up PR that adds the rest of the metadata into this pattern